spec.html

```diff
@@ -9260,7 +9260,7 @@ <h1>
           1. If |ClassHeritage| is present, return *true*; otherwise return *false*.
         1. If |ClassHeritage| is present, then
           1. If |ClassHeritage| Contains _symbol_ is *true*, return *true*.
-        1. Return the result of ComputedPropertyContains of |ClassBody| with argument _symbol_.
+        1. Return ComputedPropertyContains of |ClassBody| with argument _symbol_.
```

Comment:
This reads weirdly to me; I would much prefer to keep the previous form.

Reply:
Can you be more specific about which cases, or try to define a general rule?